pkg/maps/policymap/policymap.go

```diff
@@ -89,6 +89,19 @@ type PolicyEntryDump struct {
 	Key PolicyKey
 }
 
+// PolicyEntriesDump is a wrapper for a slice of PolicyEntryDump
+type PolicyEntriesDump []PolicyEntryDump
+
+// Less returns true if the element in index `i` is lower than the element
+// in index `j`
+func (p PolicyEntriesDump) Less(i, j int) bool {
```

Comment:
Add a unit test for this to ensure that sorting behavior isn't broken in the future.